### PR TITLE
chore/SOF-6394

### PR DESCRIPTION
**Pull request by spectre007**

### CODE REVIEW

File: lang/en/docs/properties/data/list.md [hunk 1]
```diff
@@ -416,28 +428,16 @@ The P norm measures how homogeneous a material is in terms of its chemical compo
...
 ```
 
-### [Point Group Symmetry](../../properties-directory/structural/point-group.md)
+### [Symmetry](../../properties-directory/structural/symmetry.md)
 
-The Point Group of the structure, indicating the symmetry elements that it contains, is also available. 
+The symmetry of the structure, indicating the point group and space group.
 
 ```json tab="Schema"
-{!esse/schema/properties_directory/structural/symmetry_symbol_point_group.json!}
```
Comment (timurbazhirov) [inline lang/en/docs/properties/data/list.md:424]: It's OK that we did this change for now, but we do have `symmetry_symbol_point_group` and `symmetry_symbol_space_group` as the working prototype in ESSE: https://github.com/Exabyte-io/esse/pull/198. Storing simple strings makes things easier and we should probably continue in that direction.

File: lang/en/docs/tutorials/dft/electronic/valence-band-offset.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+This tutorial employs the potential lineup method in order to determine the valence band offset, which requires the
+calculation of the macroscopically averaged electrostatic potential and valence band maximum of the two materials.
+The valence band offset for an A/B interface can then be determined via:
+$$ \Delta E_{\mathrm{VBO}} = \Delta E_{v} + \Delta V$$
+The first term, $\Delta E_{v}$, is usually referred to as the *band structure term* and defined as the difference
+of the two valence band maxima $\varepsilon_{v}$ referenced to the macroscopically averaged electrostatic potential $\overline{V}$
+in each material:
+$$ \Delta E_{v} = (\varepsilon_{v}^{A} - \overline{V}^{A}) - (\varepsilon_{v}^{B} - \overline{V}^{B}) $$
+The second term, $\Delta V$, is determined from the lineup of the macroscopically averaged electrostatic potential in
+the interface heterostructure.
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/dft/electronic/valence-band-offset.md:32]: Could we, perhaps, add some more explanation about the workflow? Maybe a link to the YAML source in WoDe.js, and a rough flowchart drawing?

Although there is a video, this page itself is rather dry without any visuals.

File: lang/en/docs/tutorials/dft/electronic/valence-band-offset.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+
+## Choose Materials
+
+When creating the job, the user needs to select **three materials** corresponding to the MoS<sub>2</sub>/WS<sub>2</sub> interface
+and the isolated monolayers of both MoS<sub>2</sub> and WS<sub>2</sub>. Each of the structures is expected to be relaxed.
+
+!!!note "Order of Materials"
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/dft/electronic/valence-band-offset.md:40]: Let's explain a bit more about these materials:

1. Where did initial structures come
2. How we relaxed these structures
3. How to find these relaxed structures in our platform, maybe even providing links to the corresponding bank material items

File: lang/en/docs/tutorials/dft/electronic/valence-band-offset.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+Before [submitting](../../../jobs/actions/run.md) the [job](../../../jobs/overview.md), the user should click on the
+["Compute" tab](../../../jobs-designer/compute-tab.md) of [Job Designer](../../../jobs-designer/overview.md) and inspect
+the [compute parameters](../../../infrastructure/compute/parameters.md) included therein.
+
+## Examine results
+
+When all [unit](../../../workflows/components/units.md) computations are complete at the end of Job execution, switching
+to the [Results tab](../../../jobs/ui/results-tab.md) of [Job Viewer](../../../jobs/ui/viewer.md) will show the results
+of the simulation, including the valence band offset as well as the plots of the planar and macroscopic average of
+the electrostatic potentials.
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/dft/electronic/valence-band-offset.md:78]: We should add picture(s) of the results here.

File: lang/en/docs/tutorials/dft/electronic/valence-band-offset.md (hunk 2 above)
Comment (spectre007) [reply lang/en/docs/tutorials/dft/electronic/valence-band-offset.md:32]: I think the YAML source would not be much help, it is very difficult to learn from it if one doesn't know how wode.js works. I made a diagram of the workflow and embed it on the tutorial page.

File: lang/en/docs/tutorials/dft/electronic/valence-band-offset.md (hunk 4 above)
Comment (spectre007) [reply lang/en/docs/tutorials/dft/electronic/valence-band-offset.md:78]: I added the macroscopically averaged potential profiles to the explanation of the potential lineup method

File: lang/en/docs/tutorials/dft/electronic/valence-band-offset.md (hunk 2 above)
Comment (timurbazhirov) [reply lang/en/docs/tutorials/dft/electronic/valence-band-offset.md:33]: Let's at least link the two? Link to docs in wode and link to wode source in here

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**